src/synapse/azext_synapse/_help.py

```diff
@@ -26,13 +26,13 @@
   - name: Create a Synapse workspace
     text: |-
         az synapse workspace create --name fromcli4 --resource-group rg \\
-          --account-url https://testadlsgen2.dfs.core.windows.net --file-system testfilesystem \\
+          --storage-account testadlsgen2 --file-system testfilesystem \\

```

Comment:
add example for storage account resource id.

Reply:
fixed

---
In reply to: [424965695](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r424965695) [](ancestors = 424965695)